src/App.js

```diff
@@ -1,25 +1,43 @@
-import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
+import AddFoodForm from './components/AddFoodForm';
+import FoodBox from './components/FoodBox';
+import SearchBar from './components/SearchBar';
+import foods from './foods.json';
+
 
 function App() {
+
+  const [food, setFood] = useState(foods);
+
+  const deleteFood = (foodName) => {
+    setFood((prevFoodArr) => {
+        const newList = prevFoodArr.filter((element) => {
+            return element.id !== foodName;
```

Comment:
`return element.id !== foodName;` I think that should be `element.name`

Reply:
Waow I was really tired...
I actually fixed this, but I committed before even testing the delete function...
Then I realised what the mistake was, fixed it. But forgot to commit again and push again...
So in conclusion, yes it was important for me to take a rest yesterday :)